Created Shell View

brandon-james-dev opened the pull request.

brandon-james-dev committed:
Create README.md

brandon-james-dev committed:
Created Shell View, adjusted App.xaml.cs to init with the Shell View selected

thecodrr reviewed (changes requested):
In Shell.xaml, the ContentPage should'nt be there. Instead, add a Frame.

brandon-james-dev committed:
Changed row size and font size for row one in Shell
Changed ContentPage in Shell to a Frame and renamed it to ContentFrame

brandon-james-dev committed:
Update README.md

Made getting acquainted with the developers for submitting pull requests a suggestion

brandon-james-dev committed:
Update README.md

Added a missing part for merging
